test/indexedDBUtils.test.js

```diff
@@ -0,0 +1,15 @@
+const indexedDBUtils = require('../src/indexedDBUtils')
+
+window.indexedDB = require('fake-indexeddb')
+
+test('DB should exist', async () => {
+  expect.assertions(1)
+  window.indexedDB.open('test1')
+  expect(await indexedDBUtils.dbExists('test1')).toBe(true)
+})
+
+test('DB should not exist', async () => {
+  expect.assertions(1)
+  window.indexedDB.open('test2')
```

Comment:
You shouldn't need to call `open('test2')` for this tests case right ?

Reply:
done